### PR TITLE
[release-v1.0] testing separation

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Do not merge - 
This is a test to run the conformance _isolated_ - if all good - a new "target" on openshift/releae will be created

### CODE REVIEW

File: openshift/e2e-tests.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ failed=0
 
 # (( !failed )) && install_tracing || failed=1
 
-(( !failed )) && run_e2e_tests || failed=1
+# (( !failed )) && run_e2e_tests || failed=1
 
 (( !failed )) && run_conformance_tests || failed=1
```
Comment (matzew) [inline openshift/e2e-tests.sh:28]: of course - this will change - after tests and moving forward with the new "goal"

**Comment by matzew** [comment]
Just conformance tests worked, 

updating to have e2e just being e2e....

**Comment by matzew** [comment]
NEXT STEP:

https://github.com/openshift/release/pull/24020

(after _THIS_ is merged, lets get that in)

**Review by pierDipi — APPROVED** [review]
/lgtm

**Pull request merged**